addons/api/addon/models/worker.js

```diff
@@ -27,15 +27,14 @@ export default class WorkerModel extends GeneratedWorkerModel {
   }
 
   /**
-   * Returns the number of config tags present on the worker.
+   * Returns the number of canonical tags present on the worker.
    * @type {number}
    */
   get tagCount() {
-    if (!this.config_tags) {
+    if (!this.canonical_tags) {
```

Comment:
The only gotcha is I'm not sure if `canonical_tags` dedupes the same tag so I'm not sure if that will match ours if we display all api and config tags separately which will show the same tags

Reply:
It would only dedupe a tag if it was set as a config and api tag right? I did some testing locally and I cannot set an api tag that is already a config tag. This may not be an issue.